pkg/query-service/app/logs/v3/json_filter.go

```diff
@@ -72,6 +73,7 @@ func getPath(keyArr []string) string {
 
 func getJSONFilterKey(key v3.AttributeKey, op v3.FilterOperator, isArray bool) (string, error) {
 	keyArr := strings.Split(key.Key, ".")
+	// i.e it should be at least body.name, and not something like body
 	if len(keyArr) < 2 {
```

Comment:
does this account for `body. ` where there is just space after the `.` ?

Reply:
keys dont contain space, it will be a invalid query

Reply:
it will be returned before reaching this point ?